assets/wire/components/WireListItem.jsx

```diff
@@ -315,6 +315,7 @@ class WireListItem extends React.Component {
                     <ListItemPreviousVersions
                         item={this.props.item}
                         isPreview={false}
+                        displayConfig={this.props.listConfig}
```

Comment:
shouldn't this be `this.props.displayConfig`?

Reply:
no, it should be a `listConfig`